packages/reports/src/Components/placeholder-component.tsx

```diff
@@ -1,8 +1,17 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import Loading from '_common/components/loading';
 
-const PlaceholderComponent = props => {
+type TPlaceholderComponent = {
+    component_icon: string;
+    empty_message_component: React.ElementType;
+    has_selected_date: boolean;
+    is_empty: boolean;
+    is_loading: boolean;
+    localized_message: string;
+    localized_period_message: string;
+};
+
+const PlaceholderComponent = (props: React.PropsWithChildren<TPlaceholderComponent>) => {
```

Comment:
```suggestion
const PlaceholderComponent = (props:TPlaceholderComponent) => {
```
@aizad-deriv The component doesn’t accept any children I guess 🤔
By the way, it would be nice if you could do object destructuring for `props` 🙏🏻